app/components/wallet-overview/wallet-overview.tsx

```diff
@@ -165,7 +167,8 @@ const WalletOverview: React.FC<Props> = ({ loading, setIsStablesatModalVisible }
               type={usdInUnderlyingCurrency ? "p3" : "p1"}
               bold={!usdInUnderlyingCurrency}
             >
-              {`~ ${usdInDisplayCurrencyFormatted}`}
+              {usdInUnderlyingCurrency && `~ `}
+              {usdInDisplayCurrencyFormatted}
```

Comment:
the logic is quite convoluted. why checking on `usdInUnderlyingCurrency` when `usdInDisplayCurrencyFormatted` is used?

the logic should also not be located in the JSX part.